src/main/java/by/onliner/utils/ClearTextFileUtil.java

```diff
@@ -0,0 +1,23 @@
+package by.onliner.utils;
+
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+
+/***
+ * The class is used to give functional of clear text file
+ */
+public class ClearTextFileUtil {
```

Comment:
What other methods can be in ClearTextFileUtil ? In my opinion this util is not extendable

Reply:
Renamed this util to FileUtil